packages/mdl-typography/README.md

```diff
@@ -0,0 +1,134 @@
+# mdl-typography
+
+MDL typography is a CSS-only component that implements the
+[Material Design typography guidelines](https://material.google.com/style/typography.html), and makes them available to
+developers as CSS classes.
+
+## Installation
+
+> Note: Installation via the npm registry will be available after alpha.
+
+
+## Usage
```

Comment:
I agree that full-blown tutorials can be a bit much but I think most (if not all) README's have a section on how to use the code.